vite.config.js

```diff
@@ -34,6 +34,20 @@ export const DEFAULT_CONFIG = {
       },
     },
   },
+  optimizeDeps: {
+    // vole-core uses a worker imported via local import (e.g. new
+    // URL('./local-worker.js', import.meta.url)). If vite bundles the worker,
+    // vole-core will not be able to find it at runtime. We exclude vole-core
+    // from dependency optimization here.
+    exclude: ["@aics/vole-core"],
+    // Have to still optimize all CommonJS dependencies of vole-core. See
+    // https://vite.dev/config/dep-optimization-options#optimizedeps-exclude
+    include: ["@aics/vole-core > tweakpane", "@aics/vole-core > geotiff", "@aics/vole-core > throttled-queue"],
```

Comment:
the exclude makes sense to me, but the includes are yucky.  Did you have to find them by trial and error?

Reply:
yup,,,, I had to look for what error messages I was getting on the console....